packages/path_provider/path_provider/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
 
 * Add iOS unit test target.
+* Remove references to the Android V1 embedding.
```

Comment:
The NEXT should become a version bump, as above.

Reply:
done